scm/driver/harness/webhook.go

```diff
@@ -236,6 +257,34 @@ func convertPushHook(dst *pushHook) *scm.PushHook {
 	}
 }
 
+func convertPullRequestCommentHook(dst *pullRequestCommentHook) *scm.PullRequestCommentHook {
+	return &scm.PullRequestCommentHook{
+		Repo: scm.Repository{
+			ID:     dst.Repo.UID,
+			Branch: dst.Repo.DefaultBranch,
+			Link:   dst.Repo.GitURL,
+			Clone:  dst.Repo.GitURL,
+		},
+		PullRequest: scm.PullRequest{
```

Comment:
**minor** can we pull out the convert for pullrequest into a separate internal method (technically even repo) to make sure we don't have to duplicate this (which risks errors)